packages/core/src/modules/oob/messages/OutOfBandInvitation.ts

```diff
@@ -121,7 +174,7 @@ export class OutOfBandInvitation extends AgentMessage {
   public readonly goal?: string
 
   public readonly accept?: string[]
-
+  @Transform(({ value }) => value.map(replaceLegacyDidSovPrefix), { toClassOnly: true })
```

Comment:
BTW just noticed this value is optional so I think we may need to do something like:

```suggestion
  @Transform(({ value }) => value?.map(replaceLegacyDidSovPrefix), { toClassOnly: true })
```

Could you maybe add a test for this (test with value both defined and undefined also with legacy dids) in `OutOfBandInvitation.test.ts`?

Reply:
I am not entirely sure how to write tests for this scenario, some direction on how to write this would be helpful if you want me to attempt to add this to this PR. @TimoGlastra

Reply:
I was able to get some assistance on this matter and believe that the test case was covered. @TimoGlastra